infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -296,8 +294,7 @@ private TxResolutionResponse txnCanCommit(TxResolutionInfo txInfo) {
             // for each key pair, check for conflict; if not present, check against the wildcard
             for (byte[] conflictParam : conflictParamSet) {
 
```

Comment:
Instead of going through the conflict window and checking for conflicts, it's better to check for `TX_ABORT_NEWSEQ,` or `TX_ABORT_SEQ_OVERFLOW` first. Otherwise, you can go through the conflict resolution and not fail, but fail on a `TX_ABORT_SEQ_OVERFLOW`. Reversing the check order can be faster when the cache window is moving fast.

Reply:
Done.